From e951bdb8b1b93e288c0147a722a22017d63e3f17 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Fri, 8 Dec 2023 15:16:39 +0000
Subject: [PATCH 0787/1085] overlays: rpi-poe: Add PWM polarity value

Newer kernels expect PWM references to include a polarity value as the
third parameter.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/rpi-poe-overlay.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/overlays/rpi-poe-overlay.dts
+++ b/arch/arm/boot/dts/overlays/rpi-poe-overlay.dts
@@ -14,7 +14,7 @@
 				compatible = "pwm-fan";
 				cooling-levels = <0 1 10 100 255>;
 				#cooling-cells = <2>;
-				pwms = <&fwpwm 0 80000>;
+				pwms = <&fwpwm 0 80000 0>;
 			};
 		};
 	};
